src/components/contextual/pages/pools/UnstakedPoolsTable.vue

```diff
@@ -1,20 +1,14 @@
 <script setup lang="ts">
-import { uniqBy } from 'lodash';
-import { computed, ref } from 'vue';
+import { computed, onMounted, ref } from 'vue';
 import { useI18n } from 'vue-i18n';
-
 import PoolsTable from '@/components/tables/PoolsTable/PoolsTable.vue';
-import useUserPoolsQuery from '@/composables/queries/useUserPoolsQuery';
-import useStaking from '@/composables/staking/useStaking';
-import { isMigratablePool } from '@/composables/usePool';
-import { bnum } from '@/lib/utils';
 import { configService } from '@/services/config/config.service';
-import { Pool, PoolWithShares } from '@/services/pool/types';
+import { Pool } from '@/services/pool/types';
 import useWeb3 from '@/services/web3/useWeb3';
-import { getAddressFromPoolId } from '@/lib/utils';
-import StakingProvider from '@/providers/local/staking/staking.provider';
-
-import StakePreviewModal from '@/components/contextual/pages/portfolio/staking/StakePreviewModal.vue';
+import { useUserPools } from '@/providers/local/user-pools.provider';
+import StakePreviewModal from '../pool/staking/StakePreviewModal.vue';
```

Comment:
Better using '@/...' import to facilitate future refactors.

Reply:
Done.